src/tests/ftest/util/ior_test_base.py

```diff
@@ -235,15 +237,19 @@ def run_ior(self, manager, processes, intercept=None, display_space=True,
                 Default is self.pool.
             out_queue (queue, optional): Pass the exception to the queue.
                 Defaults to None.
+             env (EnvironmentVariables, optional): Environment to be used
+             when running ior. Defaults to None
         """
-        env = self.ior_cmd.get_default_env(str(manager), self.client_log)
-        if intercept:
-            env['LD_PRELOAD'] = intercept
-            env['D_LOG_MASK'] = 'INFO'
-            env['D_IL_REPORT'] = '1'
-            #env['D_LOG_MASK'] = 'INFO,IL=DEBUG'
-            #env['DD_MASK'] = 'all'
-            #env['DD_SUBSYS'] = 'all'
+        if not env:
+            env = self.ior_cmd.get_default_env(str(manager), self.client_log)
+            if intercept:
+                env['LD_PRELOAD'] = intercept
+                env['D_LOG_MASK'] = 'INFO'
+                env['D_IL_REPORT'] = '1'
+
+                #env['D_LOG_MASK'] = 'INFO,IL=DEBUG'
+                #env['DD_MASK'] = 'all'
+                #env['DD_SUBSYS'] = 'all'
```

Comment:
Either have "env" as passed in here extra things to add to the env, or assume env already has the default env in and do the following, then you only need to set these once.
```suggestion
        if not env:
            env = self.ior_cmd.get_default_env(str(manager), self.client_log)
        if intercept:
            env['LD_PRELOAD'] = intercept
            env['D_LOG_MASK'] = 'INFO'
            env['D_IL_REPORT'] = '1'
            #env['D_LOG_MASK'] = 'INFO,IL=DEBUG'
            #env['DD_MASK'] = 'all'
            #env['DD_SUBSYS'] = 'all'
```

Reply:
Sounds good, implementing...